docs/supported_ops.rst

```diff
@@ -106,33 +115,35 @@ Currently, DALI supports the following operations:
       intX           uintY          intX               if X > Y
      ============== ============== ================== ========================
 
-    `T` stands for any one of the supported numerical types:
-    `bool`, `int8`, `int16`, `int32`, `int64`, `uint8`, `uint16`, `uint32`, `uint64`, `float32`, `float64`.
+    ``T`` stands for any one of the supported numerical types:
+    ``bool``, ``int8``, ``int16``, ``int32``, ``int64``, ``uint8``, ``uint16``,
+    ``uint32``, ``uint64``, ``float32``, and ``float64``.
 
-    `bool` type is considered the smallest unsigned integer type and is treated as `uint1` with respect
+    ``bool`` type is considered the smallest unsigned integer type and is treated as ``uint`` with respect
     to the table above.
 
     .. note::
-        Type promotions are commutative.
+        The promotions type is commutative.
```

Comment:
```suggestion
        Type promotion is commutative.
```
It's exactly that. "Type promotion" is the operation that is commutative - like addition or multiplication.

Reply:
Done